gensim/models/phrases.py

```diff
@@ -133,19 +133,30 @@ def __init__(self, sentences=None, min_count=5, threshold=10.0,
         `delimiter` is the glue character used to join collocation tokens, and
         should be a byte string (e.g. b'_').
 
+        `recode_to_utf8` is an optional parameter to tokenize the input words into
+        utf8 encoding. For faster Phrases, you may use `recode_to_utf8=False` if you have
+        larger RAM for this memory-time tradeoff.
```

Comment:
This is incorrect. If the sentences are already bytestrings, there is no penalty to `recode_to_utf8=False` at all. No "larger RAM" needed.